chips/cc26x2/src/radio/rfc.rs

```diff
@@ -587,16 +606,31 @@ impl RFCore {
             .is_set(CPEInterrupts::LAST_COMMAND_DONE);
         let tx_done = dbell_regs.rfcpe_ifg.is_set(CPEInterrupts::TX_DONE);
```

Comment:
In this section, it would be more concise and read better if you did something like:
```
let irq = dbell_regs.rfcpe_ifg.extract();

if irq.is_set(CPEInterrupts::TX_DONE)
...
```

Reply:
For now, I am only picking the irq's that I want specifically. Once all of the transmitter modes are implemented that we need and the protocol is set, I can be more general and service irq's as they come in. For now, I don't want to service a partial read irq if it is triggered before the read done irq for example.

Reply:
This wouldn't change the functionality of your code at all. It's purely stylistic.